internal/etos/suitestarter/suitestarter.go

```diff
@@ -670,8 +670,8 @@ func (r *ETOSSuiteStarterDeployment) suiteRunnerTemplate(templateName types.Name
                 value: /graveyard
               - name: OTEL_CONTEXT
                 value: {otel_context}
-              - name: OTEL_COLLECTOR_HOST
```

Comment:
Should be enough to just remove it though, right? Considering the etos config here: https://github.com/eiffel-community/etos/blob/main/internal%2Fetos%2Fetos.go#L401 should already be passed in

Reply:
That sounds reasonable, i'll verify it tomorrow, if it is in fact redundant i'll remove it altogether.